### ISSUE TITLE
Delete space button doesn't have effect on mouse hover (looks like an icon)

### ISSUE THREAD

**Issue by aditya-konarde** [openshiftio/openshift.io#1603]
It looks more like an icon as opposed to a clickable button.

![deletespace-mousehover](https://user-images.githubusercontent.com/6318164/33783207-c9e27366-dc81-11e7-92e5-0f6f7a7e23e6.png)

Area: UI

**Comment by joshuawilson** [comment]
This page has already been completely redone. It will get pushed to prod in the new year. Closing.

**Comment by joshuawilson** [comment]
@dlabrecq @catrobson @dgutride what are the plans for the `View all spaces` page? The new `My Spaces` page supports delete. This page is still active and this is still an issue on that page.

**Comment by catrobson** [comment]
We haven't submitted a PR for the new my spaces page as far as I know.  When we do we should remove the existing view all spaces page and close this issue.

**Comment by dlabrecq** [comment]
Yes, the my spaces page has been merged -- worked with Brian on that one. The view all spaces page still allows the user to search all spaces on the system -- there is no other way to do that.
